Traceback (most recent call last):
  File "main.py", line 50, in <module>
    trainer = Train(args)
  File "/home/leeandy/Deep_learning_lab/lab5/training.py", line 37, in __init__
    self.train_set = RetinopathyLoader("train")
  File "/home/leeandy/Deep_learning_lab/lab5/dataset/dataloader.py", line 23, in __init__
    self.weight_loss = self.__cal_we_loss()
AttributeError: 'RetinopathyLoader' object has no attribute '_RetinopathyLoader__cal_we_loss'
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    trainer = Train(args)
  File "/home/leeandy/Deep_learning_lab/lab5/training.py", line 37, in __init__
    self.train_set = RetinopathyLoader("train")
  File "/home/leeandy/Deep_learning_lab/lab5/dataset/dataloader.py", line 23, in __init__
    self.weight_loss = self.__cal_we_loss()
AttributeError: 'RetinopathyLoader' object has no attribute '_RetinopathyLoader__cal_we_loss'